qoffer-infrastructure/src/test/groovy/life/qbic/portal/qoffer2/web/controllers/CreateCustomerControllerSpec.groovy

```diff
@@ -27,16 +27,15 @@ class CreateCustomerControllerSpec extends Specification {
 
         CreateCustomerInput createCustomerInput = Mock()
         CreateCustomerController controller = new CreateCustomerController(createCustomerInput)
-
         when:
         controller.createNewCustomer(firstName, lastName, title, email, affiliations)
         then:
         1 * createCustomerInput.createCustomer({Customer customer ->
-            customer.firstName == firstName && \
-            customer.lastName == lastName && \
-            customer.title == academicTitle && \
-            customer.eMailAddress == email && \
-            customer.affiliations == affiliations
+            customer.getFirstName() == firstName && \
+            customer.getLastName() == lastName && \
+            customer.getTitle() == academicTitle && \
+            customer.geteMailAddress() == email && \
+            customer.getAffiliations() == affiliations
```

Comment:
Why did you switch to getters here?

Reply:
I find the getters more intuitive, but i can see that they are not needed in groovy. 
Good Catch with the getter though! 
Apparently groovy doesn't higher case the first letter during getter generation, when the second letter is already higher cased.